### DISCUSSION TITLE
MQTT Verbindung mit Benutzername und Passwort

### DISCUSSION THREAD

**Discussion by flashy02** [shining-man/bsc_fw#8] [Ideas]
Hallo,
habe gerade gesehen, dass ich bei der Einstellung im Web-Interface keine Möglichkeit habe MQTT-User als auch PWD einzugeben. Soll dies noch ergänzt werden? Als Basis-Sicherheit im privaten Netzwerk finde ich dies schon sinnvoll, bzw. wäre in meinem Falle auch notwendig damit ich mich mit meinem MQTT-Broker verbinden kann.
Viele Grüße
flashy02

**Comment by shining-man** [comment]
Aktuell ist tatsächlich nur eine unsichere Verbindung möglich. Ich werde es aber mit in das nächste Release aufnehmen.
Ich habe dafür einen Issue aufgemacht.
